65black/so-lite-react
# #162832600 Add Homepage Functionality

#### What does this PR do?
This PR allows a user to access the homepage when navigating to `/`. This PR also adds a 404 page for routes that do not exist.

#### Description of Task to be completed?
- add jumbotron on the homepage
- add header and footer
- add react routing
- add 404 page
- add tests for new components

#### How should this be manually tested?
- Clone this repo, run `npm install` and run `npm run dev`
- Navigate to `localhost:9000` on your browser
- You should see a Jumbotron
- Navigate to any route that doesn't exist and you should see a 404 page

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162832600

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-12-25 at 8 31 45 am" src="https://user-images.githubusercontent.com/36507972/50416197-95b22800-081f-11e9-9d50-a7bba8857200.png">
<img width="1440" alt="screenshot 2018-12-25 at 8 32 01 am" src="https://user-images.githubusercontent.com/36507972/50416223-bed2b880-081f-11e9-8712-882389366ad0.png">


#### Questions:
N/A
